run-gdb.sh

```diff
@@ -37,34 +41,20 @@ echo "ADB Location: " $ADB
 $ADB start-server
 
 case $DEVICE in
-    generic_x86)
-        TARGET_ARCH=x86
-        TARGET_TRIPLE=i686-linux-android
-        ;;
-    fugu)
-        TARGET_ARCH=x86
-        TARGET_TRIPLE=x86_64-linux-android
+    aosp_x86_64)
+        BINSUFFIX=64
+        DEVICE_NAME=generic_x86_64
         ;;
     *)
-        TARGET_ARCH=arm
-        TARGET_TRIPLE=arm-linux-androideabi
+        DEVICE_NAME=generic
```

Comment:
nit: it will be better to keep *) no change then we can use this for device debug as well.

Reply:
Unsure about your request regarding the changes made by @gabrielesvelto, but I'm reverting this one